docs/release-source/release/matchers.md

```diff
@@ -181,6 +181,10 @@ Requires the value to be a `Date` object.
 
 Requires the value to be a `Symbol`.
 
+#### `sinon.match.in(array)`
+
+Requires the value to be in the `array`.
+
 *Since `sinon@2.0.0`*
```

Comment:
The line about Sinon 2 belongs to the symbol matcher above. I'll fix it in my browser, if my crappy Moto G5 can take the load 😁